build/flutter_app.gni

```diff
@@ -63,8 +63,7 @@ template("flutter_jit_app") {
       "$flutter_core_snapshot_gen_dir/isolate_snapshot_instructions.bin"
 
   dot_packages = "$target_gen_dir/$dart_package_name.packages"
-  bundle_path = "$target_gen_dir/$bundle_name"
-  bundle_depfile = "$target_gen_dir/$bundle_name.d"
+  bundle_depfile = "$bundle_path.d"
```

Comment:
Shouldn't the depfile remain in the gen dir?

Reply:
Most ".d" files are next to the output they describe.